core/src/main/java/org/apache/druid/crypto/CryptoService.java

```diff
@@ -113,6 +113,7 @@ public CryptoService(
 
       SecretKey tmp = getKeyFromPassword(passPhrase, salt);
       SecretKey secret = new SecretKeySpec(tmp.getEncoded(), cipherAlgName);
+      @SuppressWarnings("InsecureCryptoUsage")
```

Comment:
What's insecure about it? It is bad?

If it's not bad, it'd be good to have a comment here about why it's not bad.

Reply:
error-prone warns if the ciphers are not compile-time constants that it can inspect for insecure defaults.
In our case our defaults are secure, but I can add a comment if that helps.

Reply:
I think it'd be helpful, since the suppression looks scary.

Reply:
added a comment